Add info about request source and target packages

saraycp opened the pull request:
Add info about request source and target packages and also a sentence about release if there is any.

For now, all of this is displayed in the Overview tab of the redesigned request show page.

**Before redesign:**

![Screenshot 2022-08-05 at 14-59-09 Request 2 (review)](https://user-images.githubusercontent.com/2581944/183082362-d55cb0ed-5d0f-4bb6-879e-ab12c0aac47e.png)

**After redesign:**

![Screenshot 2022-08-05 at 14-33-48 Open Build Service](https://user-images.githubusercontent.com/2581944/183082392-acee1078-5f6d-4c2f-b187-4569059eeeff.png)

hennevogel commented:
How about 

![183082392-acee1078-5f6d-4c2f-b187-4569059eeeff](https://user-images.githubusercontent.com/514785/183085091-13608443-2831-4bb2-abc7-4a94663c4cdb.png)

danidoni commented:
> How about
> 
> ![183082392-acee1078-5f6d-4c2f-b187-4569059eeeff](https://user-images.githubusercontent.com/514785/183085091-13608443-2831-4bb2-abc7-4a94663c4cdb.png)

I like it! I too wasn't sure about having the source + target below the tabs...

saraycp commented on src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml:
```diff
@@ -1,3 +1,8 @@
+%p= request_action_header(action, bs_request.creator)
+- if action[:type] == :maintenance_incident && action[:releaseproject] # Is there some release info?
```
I got rid of the maintenance part.

saraycp commented:
> How about
> 
> ![183082392-acee1078-5f6d-4c2f-b187-4569059eeeff](https://user-images.githubusercontent.com/514785/183085091-13608443-2831-4bb2-abc7-4a94663c4cdb.png)


I like the idea too. I simplified the title and moved the rest of the information under the title.

hennevogel commented:
codecov reporting hangs. [coverage job is fine](https://app.circleci.com/pipelines/github/saraycp/open-build-service/1769/workflows/e92a165a-ee3c-4971-9c33-6e865ad375d3/jobs/11855). merging!
